installer/config.yaml

```diff
@@ -27,10 +27,12 @@ tssc:
         manageSubscription: true
         # namespacePrefixes:
         #   - tssc-app
-        # RBAC:
+        authProvider: github
+        # Possible values: github, gitlab, microsoft
```

Comment:
Why `microsoft` instead of `azure` or `ms-azure`?

Reply:
Just following what we do for auth provider: https://github.com/redhat-appstudio/rhtap-cli/blob/main/installer/charts/tssc-dh/templates/app-config-content.yaml#L49